core/src/traits/Kokkos_ScheduleTrait.hpp

```diff
@@ -84,13 +84,22 @@ struct ScheduleTrait : TraitSpecificationBase<ScheduleTrait> {
                   "type of the errant tag.");
     static constexpr bool schedule_type_is_defaulted = false;
   };
-  template <class T>
-  using trait_matches_specification = is_schedule_type<T>;
+  // template <class T>
+  // using trait_matches_specification = is_schedule_type<T>;
```

Comment:
Why do we keep this commented and not removed?

Reply:
Addressed in 390ce7711